src/main/java/net/snowflake/client/core/auth/oauth/TokenResponseDTO.java

```diff
@@ -2,7 +2,9 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import net.snowflake.client.core.SnowflakeJdbcInternalApi;
 
+@SnowflakeJdbcInternalApi
```

Comment:
package scope class does not need internal annotation

Reply:
Removed